vuepress/docs/next/docs/curate/bundle-comparison.md

```diff
@@ -15,21 +15,21 @@ Some of the major improvements with docker-based bundles are:
 
 For specifications on docker-based bundles, see the [Bundle Details](bundle-details.md) page. 
 ### Bundle Comparison Table
-|Attribute | [git-based bundles](../../../v7.0/docs/curate/ecr-bundle-details.md)| [docker-based bundles](bundle-details.md)|
+|Attribute| [git-based bundles](../../../v7.0/docs/curate/ecr-bundle-details.md)| [docker-based bundles](bundle-details.md)|
 | :- | :-------------------- | :---------------------  |
 |`descriptorVersion`| v1| v5
-| Bundle Specifications |Defined with descriptor files | Built with [ent bundle CLI](../getting-started/ent-bundle.md) tool
-| Versioning |Bundle version set by Git tags |Bundle version set by Docker tags. For MFEs, the version defaults to 1.0.0 unless set in the [entando.json](bundle-details.md). 
-|Plugin Ingress| Generated by Entando or set in the plugin descriptor.yaml | Generated by Entando or customized in the [bundle descriptor](bundle-details.md#entando-bundle-conventions), entando.json 
-|Identifiers| Designated Bundle ID and Plugin ID | Bundles, MFEs, & Plugins have unique global IDs|
+| Bundle specifications |Defined with descriptor files | Built with [ent bundle CLI](../getting-started/ent-bundle.md) tool
+| Versioning |Bundle version set by Git tags |Bundle version set by Docker tags. For MFEs, the version defaults to 1.0.0 unless set in the [entando.json](bundle-details.md)
+|Plugin ingress| Generated by Entando or set in the plugin descriptor.yaml | Generated by Entando or customized in the [bundle descriptor](bundle-details.md#entando-bundle-conventions), entando.json
+|Identifiers| Designated Bundle ID and Plugin ID | Bundles, MFEs, & plugins have unique global IDs|
+|Bundle ID| Not included in component names | Suffix of bundle, plugin, widget, page and fragment names
```

Comment:
Not included in component names >> I don't think this is correct, it did have bundle Id, just not global & generated in a different way that would not necessarily be unique globally,